### DISCUSSION TITLE
Slightly different Repeated Commands behavior request

### DISCUSSION THREAD

**Discussion by agfirewire** [rxhanson/RectanglePro-Community#11] [Ideas]
I am coming over from the now defunct Optimal Layout.  Overall I am really enjoyed how Rectangles works, but some slight differences in how keyboard shortcuts behave is driving me a little nuts after years of muscle memory.

My primary ask is to have an option on "Repeated commands" to loop through the different positions of a given window size (left/right and quarters primarily) on the current display.  It would be very similar to how repeatedly pressing "First Third" shortcut with Repeated Commands set on: "move to adjacent on left/right, or cycle size on half" behaves right now. 

The way I have approached my shortcuts previously was each key represented a window size (half, quarters, horizontal 3rds, 40/60, 60/40, sixths) and repeated hitting of the key would just cycle them through each of the positions on the current monitor, and I would a dedicated key to move the window through my displays. This made it real easy to have everything I needed effectively mapped to just 8 keys; the six I mentioned previously plus full screen and next display. 

Thanks!

**Comment by rxhanson** [comment]
If I understand correctly, are you just looking for custom configuration of your repeated executions? Did you try the custom shortcuts in Rectangle Pro?

Here's a community discussion on configuring it: https://github.com/rxhanson/RectanglePro-Community/discussions/5

**Comment by agfirewire** [comment]
Ah this posts helps me understand the custom shortcuts a little better beyond just custom sizing.  Thanks!

**Comment by agfirewire** [comment]
Using the custom shortcuts I accomplished exactly what I wanted.  Thanks!
